app/javascript/components/carousel/CarouselSlide.vue

```diff
@@ -6,9 +6,11 @@
 
 <script>
 import { getInputs } from '../../helpers/focus-helpers';
+import mixinResponsive from '../../mixins/mixin-responsive';
 
```

Comment:
I'll remove them as you say - do we have a JS style guide/linter? Are we avoiding terminating semi-colons generally?

Reply:
Yeah I would say so. Where they are like this in older projects, it's because they're old projects.

We don't have anything established for this yet, but if you are able to configure one that can do this in your editor, it's well worth doing

Reply:
Cool. I was always a fan of these semi-colons but recently they just seem to be a bit of an eyesore, and an unnecessary one at that. I've removed them throughout the project.